src/template/components/output/test/index.test.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-env mocha */
+
+const chai = require('chai')
+const expect = chai.expect
+
+describe('Koop output', function () {
```

Comment:
I am using Mocha here because:

* I need to setup a test project for every `add` command test beforehand and Mocha's `beforeEach` hook is useful in such case. Tape does not provide a similar feature.

* I want to run CLI's tests and template's tests at the same time so their test frameworks need to be consistent.

I don't really a strong preference on one over another. I would use Tape in a simpler case.